docs/kubernetes/KubernetesIntegration.md

```diff
@@ -6,19 +6,31 @@ This guide will describe the configuration required to use the Calico network pl
    The Calico network plugin looks for five environment variables. If one is not set, it will assume a default value. If you need to override the defaults, you must set these variables in the environment of the _kubelet_ process.
 
 * #####ETCD_AUTHORITY
-   By default, the Calico network plugin will assume that the etcd datastore is located at `<MASTER_IP>:6666`. Setting the `ETCD_AUTHORITY` variable in your environment will direct Calico to the correct IP if your cluster is set up differently.
+   The location of the Calico etcd directory in the form `<IP>:<PORT>`
+   
+   Default: `kubernetes-master:6666`.
 
 * #####CALICOCTL_PATH
-   This plugin requires access to the `calicoctl` binary. If your binary is not located at `/usr/bin/calicoctl`, set the `CALICOCTL_PATH` environment variable to the correct path.
+   Path to the `calicoctl` binary.
+   
+   Default: `/usr/bin/calicoctl`
```

Comment:
This isn't true in the latest plugins.  By default, the plugin searches the user's $PATH for `calicoctl`